azure-pipelines.yml

```diff
@@ -73,13 +73,10 @@ stages:
             cd /numpy && \
             /opt/python/cp39-cp39/bin/python -mvenv venv && \
             source venv/bin/activate && \
-            target=\$(python3 tools/openblas_support.py) && \
-            cp -r \$target/lib/* /usr/lib && \
-            cp \$target/include/* /usr/include && \
-            python3 -m pip install ninja && \
+            python3 -m pip install ninja scipy-openblas32 && \
             python3 -m pip install -r test_requirements.txt && \
             echo CFLAGS \$CFLAGS && \
-            python3 -m pip install -v . && \
+            python3 -m pip install --config-setting "--with-openblas=32" -v . && \
```

Comment:
This doesn't look correct. There is no `--with-openblas` flag in `meson_options.txt`, and there shouldn't be I think. Like in my other comment further down, I think you can instead set `PKG_CONFIG_PATH` and point it at the correct location, and then the original `pip` invocation will work.